src/pages/Graph/SummaryPanelNode.tsx

```diff
@@ -345,6 +353,12 @@ export class SummaryPanelNode extends React.Component<SummaryPanelNodeProps, Sum
             {shouldRenderGatewayHostnames && this.renderGatewayHostnames(nodeData)}
           </>
         )}
+        {shouldRenderRank && (
+          <div>
+            <KialiIcon.Rank />
+            <span style={{ paddingLeft: '4px' }}>Rank: {nodeData.rank !== undefined ? nodeData.rank : 'N/A'}</span>
```

Comment:
I don't think there should be an N/A rank.  I think all nodes should have a rank within the [1..100] range.  If ranking by Outbound Edges and a node has 0 outbound edges, I think it should get the lowest rank, not N/A.

Reply:
Node rank is `N/A` if there is no scoring criteria selected or if the scoring criteria does not apply to that particular node. Most scoring criteria will probably apply to all nodes and the default rank/score should just be the lowest possible rather than `N/A`. However if the user disables ranking for any reason, I think it makes sense to assign `N/A` rather than the lowest rank since nothing has been scored/ranked.

> If ranking by Outbound Edges and a node has 0 outbound edges, I think it should get the lowest rank, not N/A.

Agreed, if a node has no edges it should get the lowest rank and not `N/A`. That's a bug in the current implementation that I'll fix with this change.

Reply:
Totally fine to assign N/A or some indicator that there is no rank, I just don't think the user should ever see that value.  Rank would only appear in the side panel when ranks are active.

Reply:
iirc @lucasponce made the case that if `Rank: ...` always shows up in the node summary panel it would encourage more users to take advantage of the feature. Since Rank always shows up in the side panel then we need a value for when there is no rank hence `N/A`. I personally don't have strong feelings one way or another.

Reply:
To keep rank consistent with the other options on the side panel, I'll keep it hidden whenever it is not selected instead of having `N/A` for now. We can revisit that later if need be.